### ISSUE TITLE
LLVM ERROR: out of memory with latest nightly (Windows)

### ISSUE THREAD

**Issue by dignifiedquire** [rust-lang/rust#62611]
I am trying to upgrade rust nightly in a project of mine and getting `out of memory` errors due to that, only on Windows though.

- old rust version: rustc 1.37.0-nightly (0dc9e9c10 2019-06-15)
- new rust version: rustc 1.38.0-nightly (0b680cfce 2019-07-09)

Build log of the failure on appveyor: https://ci.appveyor.com/project/dignifiedquire/deltachat-core-rust/builds/25928022

The failure occurs consistently when trying to compile the [psl](https://crates.io/crates/psl) crate.

**Comment by retep998** [comment]
Notably you are using the 32-bit toolchain. Do you also use the 32-bit toolchain on other platforms, or only on Windows?

**Comment by dignifiedquire** [comment]
You are right, I didn't realize. I think windows is the only toolchain I am consistently testing on 32bit atm.

**Comment by dignifiedquire** [comment]
After switching CI to 64bit, it seems to have fixed the issue. So it seems likely that not Windows, but 32bit is the trigger. 

Ref https://ci.appveyor.com/project/dignifiedquire/deltachat-core-rust/builds/25953216
